atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepQueueUtils.java

```diff
@@ -36,8 +36,7 @@
     public static final int MAX_CELLS_DEDICATED = 100_000;
     public static final int SWEEP_BATCH_SIZE = MAX_CELLS_DEDICATED;
     public static final int BATCH_SIZE_KVS = 1000;
-    public static final long READ_TS = 1L;
-    public static final long WRITE_TS = 0L;
+    public static final long READ_TS = Long.MAX_VALUE;
```

Comment:
This should be safe, since `(timestampPartition, timestampModulus, writeIndex)` which feature in the row/column components is a unique triple, so nothing should ever get written more than once